### ISSUE TITLE
secretParamName and idParamName option are not working

### ISSUE THREAD

**Issue by tiaod** [lelylan/simple-oauth2#134]

**Comment by jonathansamines** [comment]
Which version of simple-auth are you using? Do you have a reproduction on this repo I  could take a look at?

**Comment by tiaod** [comment]
@jonathansamines I'm using the latest version of `simple-auth`（"simple-oauth2": "^1.2.0"）
Here is an example, I set the `idParamName` to `"appid"`, but it still generate url using the param name "client_id"

```javascript
const credentials = {
  client: {
    id: '<client-id>',
    secret: '<client-secret>',
    secretParamName:'secret',
    idParamName:'appid'
  },
  auth: {
    tokenHost: 'https://api.oauth.com'
  }
};

// Initialize the OAuth2 Library
const oauth2 = require('simple-oauth2').create(credentials);
const authorizationUri = oauth2.authorizationCode.authorizeURL({
  redirect_uri: 'http://localhost:3000/callback',
  scope: '<scope>',
  state: '<state>'
});

console.log(authorizationUri)
```

console：

> https://api.oauth.com/oauth/authorize?redirect_uri=http%3A%2F%2Flocalhost%3A3000%2Fcallback&scope=%3Cscope%3E&state=%3Cstate%3E&response_type=code&client_id=%3Cclient-id%3E
[Finished in 0.519s]

**Comment by jonathansamines** [comment]
@tiaod Thanks for the response. I just created the fix for that. The authorizeURL was not taking into account the custom param configured at the simple oauth instance. Will release as soon.

**Comment by jonathansamines** [comment]
Solved at 1.3.0
